### DISCUSSION TITLE
About development-strategy

### DISCUSSION THREAD

**Discussion by MishaShevchenko** [HYF-Class22/group1-hyf-rebuild#34] [General]
Lishan Look, in your development strategy you shouldn't think about only the user. If you need to add icons, for example, you must describe this process. I add an a tag and indicate the path to it. In general, you can look at our last project.
Title
As a user, I have to see the 'title'
add header with 'h1'

align the header to the center of the page

**Comment by Lishan6** [comment]
OK thank you Misha

**Reply by younes8888** [reply]
The user stories remain unchanged, but you can add the HTML and CSS input.

**Comment by Lishan6** [comment]
![image](https://github.com/HYF-Class22/group1-hyf-rebuild/assets/149688695/1757bd6a-6097-4232-b57f-1727b2754c01)

**Reply by Lishan6** [reply]
![image](https://github.com/HYF-Class22/group1-hyf-rebuild/assets/149688695/9e1ca102-277c-4edb-bde1-5e9b450139f6)

**Reply by Lishan6** [reply]
and can you check it before i send pull request thanks
